OneSignalSDK/onesignal/src/main/java/com/onesignal/LocationGMS.java

```diff
@@ -223,11 +231,13 @@ else if (permissionList.contains("android.permission.ACCESS_COARSE_LOCATION")) {
                   sendAndClearPromptHandlers(promptLocation, true);
                   startGetLocation();
                } else {
-                  sendAndClearPromptHandlers(promptLocation, false);
+                  String alertDialogTitle = context.getString(R.string.location_not_available_title);
+                  String alertDialogMessage = context.getString(R.string. location_not_available_message);
```

Comment:
You use these `Strings` in two places just move them up top outside of the first if statement

Reply:
Remove space `R.string. location_not_available_message);`
`R.string.location_not_available_message);`